internal/swagger/swagger.go

```diff
@@ -193,13 +192,13 @@ type ListSubscriptionsParameters struct {
 
 	// The sort field to use for the listing
 	//
-	// enum: username,start-date,end-date
+	// enum: ["username","start-date","end-date"]
```

Comment:
The way to specify enumeration constants in `go-swagger` changed recently, so we were getting some warnings when we generated the specification.